UPGRADE.md

```diff
@@ -1,5 +1,18 @@
 # Upgrade Instructions
 
+## Upgrade from 3.x to 4.x
+* We changed the signature of `NewStringRule` method, instead of getting `validator function` and `message`, it now
+get `validator function` and `translationKey`, this is because of adding the translation feature. The following snippet shows
+how to modify your code if you want to define new string rule:
+ ```go
+// 3.x
+// Assume Email is your custom rule:
+var Email = validation.NewStringRule(govalidator.IsEmail, "must be a valid email address")
+
+// 4.x
+var Email = validation.NewStringRule(govalidator.IsEmail, "email").DefaultMessage("must be a valid email address")
+```
+
```

Comment:
I'd like to remain BC by introducing a new function `NewStringValidator(validator stringValidator, code, message string) StringRule`.  We can deprecate the old function, but not removing/replacing it. This avoids the trouble of upgrading, considering the fact that the main use scenario of this library doesn't require i18n.

Reply:
Right, I mark the old signature as deprecated and define `NewStringValidator` function.